pkg/ephemeral-disk/ephemeral-disk.go

```diff
@@ -36,7 +37,7 @@ const (
 
 type EphemeralDiskCreatorInterface interface {
 	CreateBackedImageForVolume(volume v1.Volume, backingFile string) error
-	CreateEphemeralImages(vmi *v1.VirtualMachineInstance, isBlockVolume map[string]bool) error
+	CreateEphemeralImages(vmi *v1.VirtualMachineInstance, domain *api.Domain) error
```

Comment:
just a nit: we can keep passing the `domain` as a pointer, since it's quite a large object.

Reply:
Very good catch! Always a fan of tiny efficiencies. - https://github.com/kubevirt/kubevirt/pull/6237/commits/bfcd542c4226c3720cc1a8059af109c6474e137a